### PR TITLE
Enable logging within integration tests

### PR DESCRIPTION
**Pull request by mikedamay**

### CODE REVIEW

File: src/Exercism.Representers.CSharp/Options.cs [hunk 1]
```diff
@@ -13,7 +13,12 @@ internal class Options
         [Value(2, Required = true, HelpText = "The directory to which the results will be written")]
         public string OutputDirectory { get; }
 
-        public Options(string slug, string inputDirectory, string outputDirectory) =>
-            (Slug, InputDirectory, OutputDirectory) = (slug, inputDirectory, outputDirectory);
+        [Option('s', "SuppressLogConfiguration", Required = false, HelpText = "A flag (used for unit testing) to prevent configuration of the logger")]
+        public bool SuppressLogConfiguration { get; }
```
Comment (ErikSchierboom) [inline src/Exercism.Representers.CSharp/Options.cs:17]: Naming wise, I think I would prefer `SkipLogConfiguration`, but that is just a personal preference :)

**Review by ErikSchierboom — APPROVED** [review]

File: src/Exercism.Representers.CSharp/Options.cs (hunk 1 above)
Comment (mikedamay) [reply src/Exercism.Representers.CSharp/Options.cs]: Changed

**Comment by mikedamay** [comment]
@ErikSchierboom BTW I don't have permissions to merge into master

**Review by ErikSchierboom — APPROVED** [review]

**Comment by mikedamay** [comment]
@ErikSchierboom Please merge this when you get a chance.  I don't have the juice.

**Comment by ErikSchierboom** [comment]
Done

**Pull request merged**